tap_core/src/tap_manager/manager.rs

```diff
@@ -135,6 +135,27 @@ impl<
         Ok(())
     }
 
+    /// Removes all receipts that have a timestamp less or equal than the last RAV timestamp_max.
```

Comment:
I think this comment could point out that the receipts being deleted are now obsolete since they are aggregated into the stored RAV. That will allow the user to know what they are getting from this function without needing to understand the protocol.